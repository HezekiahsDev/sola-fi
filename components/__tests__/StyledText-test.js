import * as React from "react";
import renderer, { act } from "react-test-renderer";

import { MonoText } from "../StyledText";

jest.mock("../useColorScheme", () => ({
  useColorScheme: () => "light",
}));

it(`renders correctly`, () => {
  let rendered;

  act(() => {
    rendered = renderer.create(<MonoText>Snapshot test!</MonoText>);
  });

  expect(rendered.toJSON()).toMatchSnapshot();

  act(() => {
    rendered.unmount();
  });
});
