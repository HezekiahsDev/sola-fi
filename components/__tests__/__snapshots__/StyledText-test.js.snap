// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#1A1A1A",
      },
      [
        undefined,
        {
          "fontFamily": "SpaceMono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
